images/build-integration/entrypoint.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+# Vars
+BINARY=integration.test
+if [[ ${PLATFORM} == 'windows' ]]; then
+    BINARY=integration.test.exe
+fi
+BINARY_PATH="/opt/crc/bin/${PLATFORM}-amd64/${BINARY}"
+
+# Results
+RESULTS_PATH="${RESULTS_PATH:-/output}"
+
+
+if [ "${DEBUG:-}" = "true" ]; then
+    set -xuo 
+fi
+
+# Validate conf
+validate=true
+[[ -z "${TARGET_HOST+x}" ]] \
```

Comment:
@adrianriobo what is the use of this `+x` suffix here and is there any reason we are using `#!/bin/sh` instead `bash` shell, doesn't it part of container?

Reply:
to be honest, bash is part of ubi images, but the intention is not limit the script to ubi, as an example alpine only brings sh , did not check but distroless probably the same...so I would prefer to use sh

Reply:
about the +x I got some errors when testing this and the variable were unset, got errors on the condition evaluation... so this would sanitize it a little bit